File: Punto.py
import math

class Punto:
    def __init__(self, x , y):
        self.X = x
        self.Y = y
    def coord_cartesiana(self):
        return "Coordenadas {} {}".format(self.X, self.Y)
    def coord_polares(self):
        p1 = ((self.X)**2 + (self.Y)**2)**(1/2)
        p2 = math.degrees(math.atan2(self.Y, self.X))
        return "Coordenadas polares {} {:.2f}".format(p1,p2)
    def __str__(self):
        return f"Punto {self.X , self.Y}"


p = Punto(3, 4)
print(p.coord_cartesiana())
print(p.coord_polares())
print(p)

File: Punto.java
package Punto;

class Punto {
	public float x;
	public float y;
	
	public Punto(float X, float Y) {
		x = X;
		y  = Y;
	}
	
	public String coord_cartesianas(){
		return "Cordenadas Cartesianas ("+ x +" , "+ y +")";
	}
	public String coord_polares() {
		double p1 = Math.sqrt(x * x + y * y);
		double p2 = Math.toDegrees(Math.atan2(y, x));
		return "Coordenadas polares ("+p1+" , "+p2+")";
	}
	public String toString(){
		return "punto ("+x+" , "+y+")";
		
	}
	public static void main(String[]args) {
		Punto p = new Punto(3, 4);
		System.out.println(p.coord_cartesianas());
		System.out.println(p.coord_polares());
		System.out.println(p.toString());
	}
}
